unicodetools/src/test/java/org/unicode/text/UCD/TestCodeInvariants.java

```diff
@@ -115,10 +116,10 @@ public static int testScriptExtensions() {
                 System.out.println("Script Extensions invariant works for version " + age + "\n");
             }
 
-        return testResult;
+        assertEquals(TEST_PASS, testResult, "Invariant test for Script_Extensions failed!");
     }
 
-    public static int testGcbInDecompositions(boolean showAllNfds) {
+    public void testGcbInDecompositions() {
```

Comment:
Missed one test annotation...
```suggestion
    @Test
    public void testGcbInDecompositions() {
```

Reply:
done

Reply:
I keep dropping that… what do i have against GCB decompositions?!